lib/commands/app-management.js

```diff
@@ -71,6 +72,37 @@ commands.mobileActivateApp = async function mobileActivateApp (opts = {}) {
   return await this.proxyCommand('/wda/apps/activate', 'POST', requireOptions(opts, ['bundleId']));
 };
 
+/**
+ * Kill the given bundle id process via instruments service.
+ * https://github.com/YueChen-C/py-ios-device/blob/51f4683c5c3c385a015858ada07a5f1c62d3cf57/ios_device/cli/base.py#L220
+ *
+ * @param {Object} opts - Options set, which must contain `bundleId` property
+ * @returns {boolean} Returns true if the bundle id process was killed. Otherwise false.
+ */
+commands.mobileKillApp = async function mobileKillApp (opts = {}) {
+  if (!this.isRealDevice()) {
+    throw new errors.UnsupportedOperationError('A real device is required');
+  }
+
+  if (!this.opts.device.udid) {
```

Comment:
I assume opts.device.udid should be always defined for real devices

Reply:
Yes, right